scripts/e2e-ci.sh

```diff
@@ -79,125 +82,140 @@ task() {
+
+    elif [ "$LARGE" -eq 1 ]; then
+        step Initalizing ilab with l40s-x4 train profile
+        ilab config init --non-interactive --train-profile="${SCRIPTDIR}/test-data/train-profile-l40s-x4.yaml"
+
+        step Setting large-size SDG-specific config
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" generate.pipeline full
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" generate.teacher.model_path "${CACHE_HOME}/instructlab/models/${MIXTRAL_8X7B_MODEL}"
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" generate.teacher.vllm.gpus 4
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" generate.teacher.vllm.llm_family mixtral
+
+        step Setting large-size Training-specific config
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" train.ckpt_output_dir "${CACHE_HOME}/instructlab/checkpoints"
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" train.phased_mt_bench_judge "${CACHE_HOME}/instructlab/models/${PROMETHEUS_8X7B_MODEL}"
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" train.model_path "${CACHE_HOME}/instructlab/models/${GRANITE_7B_MODEL}"
+
+        step Setting large-size Eval-specific config
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" evaluate.gpus 4
+        python "${SCRIPTDIR}"/e2e_config_edit.py  "${CONFIG_HOME}/instructlab/config.yaml" evaluate.mt_bench.judge_model "${CACHE_HOME}/instructlab/models/${PROMETHEUS_8X7B_MODEL}"
     fi
-    task ilab Initializing Complete
+    task InstructLab initialization Complete
+}
+
+test_config_show() {
+    task Output active config
+    ilab config show
+    task Config show Complete
 }
 
 test_download() {
     task Download models
-    if [ "$LARGE" -eq 1 ]; then
+    if [ "$MEDIUM" -eq 1 ]; then
+        # TODO
+        step Downloading the TODO model as the teacher model for SDG
+        step Downloading the TODO model as judge model for evaluation
+        step Downloading the TODO model to train
+    elif [ "$LARGE" -eq 1 ]; then
         step Downloading the mixtral 8x7b instruct model as the teacher model for SDG
         ilab model download --repository ${MIXTRAL_8X7B_MODEL} --hf-token "${HF_TOKEN}"
         step Downloading the prometheus-8x7b model as judge model for evaluation
         ilab model download --repository ${PROMETHEUS_8X7B_MODEL} --hf-token "${HF_TOKEN}"
         step Downloading granite-7b-lab model to train
-        ilab model download --repository instructlab/granite-7b-lab
+        ilab model download --repository ${GRANITE_7B_MODEL}
     fi
-    task Downloading Models Complete
+    task Downloading models Complete
 }
 
 test_list() {
     task List the Downloaded Models
-
-    if [ "$LARGE" -eq 1 ]; then
```

Comment:
Why were these greps removed? They test to make sure that the downloaded models are listed properly. Is it because other tests that run more frequently test model listing the same?

Reply:
The greps don't really do anything and won't even fail out if the value isn't there. Better just to see the full command output.

Reply:
After talking we're gonna keep em.